particles/PipeApps/source/RenderNotification.js

```diff
@@ -13,21 +13,12 @@
 defineParticle(({DomParticle, html, log}) => {
 
   return class extends DomParticle {
-    // for testing under DOM modality
-    get template() {
-      return `<span>{{json}}<span>`;
-    }
-    render(props, state) {
-      const json = JSON.stringify({
+    setHandles(handles) {
```

Comment:
please, use UiParticle base class instead of DomParticle (it shouldn't affect implementation otherwise, just the base class)